### PR TITLE
[docker-sonic-mgmt] Merge spytest dependencies into mgmt docker

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

**- How to verify it**
Build `docker-sonic-mgmt` and check that the proper dependencies are there.

**- Description for the changelog**

Merge spytest dependencies into mgmt docker

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by daall** [comment]
retest vsimage please

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -24,8 +24,9 @@ RUN apt-get install -y \
                 cmake \
                 tcpdump \
                 snmp \
-                python-dev \
-                python-scapy
+                python-scapy \
```
Comment (tahmed-dev) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:27]: is python-dev not required anymore?

Comment (daall) [reply dockers/docker-sonic-mgmt/Dockerfile.j2:27]: This was a duplicate.

**Comment by yxieca** [comment]
retest vsimage please

**Comment by daall** [comment]
retest vsimage please

**Pull request merged**